request.go

```diff
@@ -322,6 +322,16 @@ func (r *Request) SetFileReader(param, fileName string, reader io.Reader) *Reque
 	return r
 }
 
+// SetMultipartFormData method allows simple form data to be attached to the request as `multipart:form-data`
+func (r *Request) SetMultipartFormData(data map[string]string) *Request {
```

Comment:
@ArcticSnowman Technically, the method `r.SetMultipartField` could be used directly for the described use case on #336. So the new method `SetMultipartFormData` is decorated/helper.

Reply:
Yeah it's totally a helper function just as a convenience